azure-pipelines.yml

```diff
@@ -7,15 +7,15 @@ resources:
       endpoint: github
 
 jobs:
-  - template: nsls2-collection-2021-2.2.yml@templates
+  - template: 2022-2.2-py39-tiled.yml@templates
     parameters:
       beamline_acronym: HXN
-  - template: nsls2-collection-2021-2.2-py39.yml@templates
+  - template: 2022-2.2-py38-tiled.yml@templates
     parameters:
       beamline_acronym: HXN
-  - template: nsls2-collection-2021-3.0-py37.yml@templates
+  - template: 2022-2.3-py39-tiled.yml@templates
     parameters:
       beamline_acronym: HXN
-  - template: nsls2-collection-2021-3.0-py39.yml@templates
+  - template: 2022-2.3-py38-tiled.yml@templates
     parameters:
       beamline_acronym: HXN
```

Comment:
@mrakitin I am not sure it is important. But the tests are still not passing. There is some kafka issue. Do you know how to fix it by any chance?

Reply:
Did it use to work with the previous conda envs?

Looks like this code is failing - https://github.com/NSLS-II-HXN/profile_collection/blob/e16ba6f4befce354473448b5bac8cde1efb611e9/startup/00-startup.py#L70-L87

We may need to stand up the kafka server in the CI to have this custom (not coming via `nslsii.configure_base(...)`) code to pass. The beamlines who configure it via nslsii have the failures tolerated if it can't connect in the CI. Or we can do it conditionally, and skip it in the CI.